tools/cockpit.spec

```diff
@@ -124,6 +124,7 @@ if [ -n "%{patches}" ]; then
 	git add -f . && git commit -a -q -m "Base" && git tag -a initial --message="initial"
 	git am --whitespace=nowarn %{patches}
 	touch -r $(git diff --name-only initial..HEAD) .git
+	touch configure.ac aclocal.m4 configure Makefile.am Makefile.in
```

Comment:
I resolved this now with
```
touch -r $(git diff --name-only initial..HEAD) .git aclocal.m4
```
and adding a manual patch (just comment) to `Makefile.am`. Thanks @martinpitt 

The original proposal here is dangerous, since it can hide actual patch errors.